src/explore-education-statistics-common/src/components/Tabs.tsx

```diff
@@ -66,6 +67,9 @@ const Tabs = ({ children, id }: Props) => {
                 onClick={event => {
                   event.preventDefault();
                   setSelectedTab(index);
+                  if (typeof onToggle === 'function') {
+                    onToggle(props.title);
```

Comment:
Don't pass through just the title, this does not necessarily help you identify which tab has actually been clicked. For example, what if there are two tabs that accidentally have the same title?

You should at least pass through the section's ID as well in an object e.g.

```javascript
onToggle({
  id: sectionId,
  title: props.title,
});
```